repos/system_upgrade/el7toel8/actors/initramdiskgenerator/libraries/initramgen.py

```diff
@@ -68,11 +91,14 @@ def generate_initram_disk(context):
     copy_dracut_modules(context, modules)
     context.copy_to(generator_script, os.path.join('/', INITRAM_GEN_SCRIPT_NAME))
     install_initram_deps(context)
+    install_files = _install_files(context)
     context.call([
         '/bin/sh', '-c',
```

Comment:
FYI, based on a discussion with @AloisMahdal, this style of calling shell script is not that safe. I created an issue to deal with that later: https://github.com/oamg/leapp-repository/issues/376

Reply:
Agree that this is not ideal solution. We should fix it later in different PR as this one is starting to be pretty big.

Reply:
Sure, let's fix it later in a separate PR. That was also the reason why I created the issue.